### ISSUE TITLE
Wheel in Mame

### ISSUE THREAD

**Issue by Pierro21** [sselph/scraper#166]
Hi, 
First congratulation for your work, it's really pleasent to have a good scraper!
Second, I noticed that for the console roms, the "marquee" is actually the wheel image. Is there a possibility to do a similar option for that for the mame roms? As the marquee's size change every time, it's easier to integrate the wheel in theme!

I can help if you want, I just don't understand yet how you pick images from ss. If you can light me up I will be please to help!

Can't wait to hear from you!

**Comment by sselph** [comment]
Yeah I've done a bad job of syncing the image types across sources. So in thegamesdb they have a clearlogo which I think is similar to the wheel in ss. Which I think I mapped to marquee because I didn't what a wheel was supposed to be. So maybe remaping wheel to logo instead of marquee. That leaves the -marquee flags which would actually be using the logo and possibly fallback to marquee. This is how this all works:

This code parses the API response:
https://github.com/sselph/scraper/blob/master/ss/ss.go#L97

This code maps the API data to a standard image type:
(ss console) https://github.com/sselph/scraper/blob/master/ds/ss.go#L225
(ss mame) https://github.com/sselph/scraper/blob/master/ds/ss_mame.go#L53

Then this code pulls the image and downloads it and puts it in the XML:
https://github.com/sselph/scraper/blob/master/rom/rom.go#L493

I don't mind fixing it or you can if you like but I would definitely want to hear your ideas on how to best handle the images.
